template/READ_BEFORE_COMMIT.md.tpl

```diff
@@ -0,0 +1,8 @@
+# Instructions
+
+This document lays out the next steps for AWS service teams upon bootstrapping the ACK {{ .ServicePackageName }}-controller repository successfully.
+
+1. To get started, look inside the `generator.yaml` file, and remove the resource(s) from the ignore list and execute `make build-controller` from the ACK code-generator.
```

Comment:
include instruction to update full_name and short_name inside metadata.yaml

Reply:
The instruction to update full_name and short_name inside metadata.yaml has been added. Thank you.